src/main/java/com/gluonhq/substrate/target/LinuxTargetConfiguration.java

```diff
@@ -86,18 +84,37 @@ public boolean link(ProcessPaths paths, ProjectConfiguration projectConfiguratio
             "com.sun.javafx.font.freetype.FT_Glyph_Metrics");
 
     @Override
-    List<String> getJNIClassList (boolean usejavafx) {
-        List<String> answer = super.getJNIClassList(usejavafx);
-        if (usejavafx) answer.addAll(javafxJNILinuxClassList);
+    List<String> getJNIClassList(boolean useJavaFX, boolean usePrismSW) {
+        List<String> answer = super.getJNIClassList(useJavaFX, usePrismSW);
+        if (useJavaFX) answer.addAll(javafxJNILinuxClassList);
         return answer;
     }
 
     @Override
-    List<String> getTargetSpecificLinkFlags(boolean usejavafx) {
+    List<String> getTargetSpecificLinkFlags(boolean useJavaFX, boolean usePrismSW) {
         List<String> answer = new LinkedList<>();
         answer.add("-rdynamic");
-        if (!usejavafx) return answer;
+        if (!useJavaFX) return answer;
+
+        ProcessBuilder process = new ProcessBuilder("pkg-config", "--libs", "gtk+-3.0", "gthread-2.0", "xtst");
+        process.redirectErrorStream(true);
+        try {
+            Process start = process.start();
+            InputStream is = start.getInputStream();
+            start.waitFor();
+            BufferedReader br = new BufferedReader(new InputStreamReader(is));
+            String line;
+            while ((line = br.readLine()) != null) {
+                Logger.logInfo("[SUB] " + line);
```

Comment:
why "[SUB]"?

Reply:
In general, why would you print this? This will be confusing to the developer (especially the output of pkg-config)

Reply:
Leftover, I added it to test Travis, and thanks to the messages I figured out what packages where missing. So probably it is worth keeping (with logDebug, so it will printed to the log file).